HW_Lection4/Task2_3.js

```diff
@@ -0,0 +1,23 @@
+// 3.
+// Преобразовать Task 2 - 1 таким образом, чтобы значение НАПРИМЕР '2' (т.е. ЛЮБАЯ строка в которой лежат ТОЛЬКО ЦИФРЫ) пропускалось, 
+//   преобразовываясь в number
+
+let age = '10';
+let age_2 = 18;
+let age_3 = 61;
+
+if (typeof age === 'number') {
```

Comment:
```suggestion
if (!isNaN(age)) {
```

Add output to `else` block for this case